.github/workflows/release.yaml

```diff
@@ -33,10 +33,10 @@ jobs:
         run: |
           version="${{ github.ref_name }}"
           version=${version#v}
-          poetry version ${{ github.ref_name }}
+          uvx --from=toml-cli toml set --toml-path=pyproject.toml project.version "$version"
```

Comment:
uv has no function to set the project version, so it needs to be updated in `pyproject.toml` manually.